src/sbgnStyle/glyph/entityPoolNodeShapes.js

```diff
@@ -0,0 +1,122 @@
+const baseShapes = require('./baseShapes.js');
+const svgStr = require('../util/svg.js');
+
+const entityPoolNodes = {
+
```

Comment:
Some copypasta in this file and the process nodes file.

Most of the duplication is through the declaration of this stylemap.  Might be worth it to think of ways to reduce this repeated code.  (default arguments?, rest parameters?)